src/System.Private.CoreLib/shared/System/Runtime/CompilerServices/AsyncValueTaskMethodBuilder.cs

```diff
@@ -12,25 +12,25 @@ namespace System.Runtime.CompilerServices
     [StructLayout(LayoutKind.Auto)]
     public struct AsyncValueTaskMethodBuilder
     {
-        /// <summary>The <see cref="AsyncTaskMethodBuilder"/> to which most operations are delegated.</summary>
-        private AsyncTaskMethodBuilder _methodBuilder; // mutable struct; do not make it readonly
-        /// <summary>true if completed synchronously and successfully; otherwise, false.</summary>
-        private bool _haveResult;
-        /// <summary>true if the builder should be used for setting/getting the result; otherwise, false.</summary>
-        private bool _useBuilder;
+        /// <summary>The lazily-initialized built task.</summary>
+        private Task<VoidTaskResult> m_task; // lazily-initialized: must not be readonly. Debugger depends on the exact name of this field.
```

Comment:
If it's no longer a mutable struct, the "// lazily-initialized: must not be readonly." comment isn't necessary.